ceph.spec.in

```diff
@@ -179,6 +179,7 @@ BuildRequires:	xfsprogs
 BuildRequires:	xfsprogs-devel
 BuildRequires:	xmlstarlet
 BuildRequires:	nasm
+BuildRequires:  lua-devel
```

Comment:
How do we ensure this is Lua 5.3?

(Makes me wonder why not use Lua 5.4... but I suppose that's not available on debian/centos yet?)

Reply:
CentOS/RHEL8 only has lua 5.3 packaged. see https://centos.pkgs.org/8/centos-powertools-x86_64/lua-devel-5.3.4-11.el8.x86_64.rpm.html

> How do we ensure this is Lua 5.3?

could put
```
BuildRequires:  lua-devel >= 5.3
```

but i think `BuildRequires:  lua-devel` is good enough.